wsd/DocumentBroker.cpp

```diff
@@ -434,12 +434,18 @@ void DocumentBroker::pollThread()
         // Remove idle documents after 1 hour.
         if (isLoaded() && getIdleTimeSecs() >= IdleDocTimeoutSecs)
         {
-            // Stop if there is nothing to save.
-            LOG_INF("Autosaving idle DocumentBroker for docKey [" << getDocKey() << "] to kill.");
-            if (!autoSave(isPossiblyModified()))
+            // Don't hammer on saving.
+            if (std::chrono::duration_cast<std::chrono::seconds>(now - _lastSaveRequestTime).count()
```

Comment:
yes, that a good thing, previously exactly this one caused high CPU usage I think